pkg/policy/rule.go

```diff
@@ -17,7 +17,9 @@ package policy
 import (
 	"fmt"
 
+	"github.com/cilium/cilium/pkg/ip"
 	"github.com/cilium/cilium/pkg/policy/api"
+	"net"
```

Comment:
Move this import into the other group above.

Reply:
Fixed.